# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1521423784 25200
#      Sun Mar 18 18:43:04 2018 -0700
# Node ID 2f6698dd7dadd4a8449123d8b52541b78331a060
# Parent  a73dedb0d9b59e2da411037fb4d23a9c074d6996
# Parent  8862b95d64ec1b1cdae7c70cc6c6a4db29f6b672
maint: merge stable to default

diff --git a/doc/interpreter/module.mk b/doc/interpreter/module.mk
--- a/doc/interpreter/module.mk
+++ b/doc/interpreter/module.mk
@@ -349,16 +349,17 @@ doc_EXTRA_DIST += \
   %reldir%/graphics_properties.mk \
   %reldir%/images \
   %reldir%/images.awk \
   %reldir%/images.mk \
   %reldir%/macros.texi \
   %reldir%/mk-doc-cache.pl \
   %reldir%/mkcontrib.awk \
   %reldir%/munge-texi.pl \
+  %reldir%/prepare_qhelp.py \
   $(DOC_IMAGES) \
   $(DOC_IMAGES_SRC) \
   $(LOGOS) \
   $(TXI_SRC)
 
 doc_MAINTAINERCLEANFILES += \
   AUTHORS \
   $(BUILT_DOC_IMAGES) \
